models/cont_delay_synapse.h

```diff
@@ -45,8 +45,8 @@ Description
 
 cont_delay_synapse relaxes the condition that NEST only implements delays
 which are an integer multiple of the time step h. A continuous delay is
-decomposed into an integer part (delay_) and a double (delay_offset_) so
-that the actual delay is given by  delay_*h - delay_offset_. This can be
+decomposed into an integer part (delay\_) and a double (delay_offset\_) so
+that the actual delay is given by  delay_*h - delay_offset\_. This can be
```

Comment:
Not exactly sure what is meant here, but if the underscore is code maybe like this (?)
```suggestion
decomposed into an integer part (``delay_``) and a double (``delay_offset_``) so
that the actual delay is given by  ``delay_*h - delay_offset_``. This can be
```